prow/cluster/monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -11,6 +11,7 @@ stringData:
         module: [http_2xx]
       static_configs:
         - targets:
+          # ATTENTION: Keep this in sync with the list in mixins/prometheus/prober_alerts.libsonnet
```

Comment:
NIP: we could use jsonnet to generate this yaml file too.
The targets will be defined as a list var and used by the 2 jsonnet files (and no need for the attention): [example](https://github.com/openshift/release/blob/00ff806f2d2295db7496fc9d4bbe02ad1c96af87/cluster/ci/monitoring/mixins/config.libsonnet#L13)

The missing part is that jsonnet binary (also the jsonnet bazel rule) can convert json to yaml, with _restrictions_: https://github.com/bazelbuild/rules_jsonnet/issues/114

And then this comment became a NIP. :)

If you know a bazel rule which can convert json to yaml, I can take care of the rest.

Reply:
/shrug